test/integration/scheduler/preemption_test.go

```diff
@@ -65,7 +65,7 @@ func waitForNominatedNodeNameWithTimeout(cs clientset.Interface, pod *v1.Pod, ti
 		}
 		return false, err
 	}); err != nil {
-		return fmt.Errorf("Pod %v/%v annotation did not get set: %v", pod.Namespace, pod.Name, err)
+		return fmt.Errorf("pod %v/%v annotation did not get set: %v", pod.Namespace, pod.Name, err)
```

Comment:
IIRC Pod is deliberately used in some places instead of pod (found this: https://github.com/kubernetes/website/issues/20862)

Reply:
That's really a trick issue. It's very true that when documenting, it's recommended to use Pod/Node to represent its a particular (Kubernetes) term. In terms of error strings, they're sort of conflicted.

I'm more inclined to keep using Pod/Node, even when they're the first word in the error string. WDYT?

Reply:
The change for Pod/Node have been restored.